### PR TITLE
publish consumer lag as part of ticker

### PR DESCRIPTION
**Pull request by vnktram**

### CODE REVIEW

File: internal/subscriber/ordering.go [hunk 1]
```diff
@@ -43,6 +43,14 @@ func (s *OrderedImplementation) GetSubscription() *subscription.Model {
 	return s.subscription
 }
 
+// GetConsumerLag returns perceived lag for the gievn Subscriber
```
Comment (sakshi-goyal-razorpay) [inline internal/subscriber/ordering.go:46]: typo

File: internal/subscriber/ordering.go [hunk 2]
```diff
@@ -43,6 +43,14 @@ func (s *OrderedImplementation) GetSubscription() *subscription.Model {
...
+func (s *OrderedImplementation) GetConsumerLag() map[string]uint64 {
+
+	lag, _ := s.consumer.GetConsumerLag(s.ctx)
```
Comment (sakshi-goyal-razorpay) [inline internal/subscriber/ordering.go:49]: why are we ignoring err here..?

File: internal/subscriber/ordering.go (hunk 1 above)
Comment (vnktram) [reply internal/subscriber/ordering.go]: Fixed

**Comment by sakshi-goyal-razorpay** [comment]
Can we also write an integration test inside tests/integration/messagebroker/messagebroker_kafka_test.go..?

**Comment by vnktram** [comment]
@sakshi-goyal-razorpay Added integration test. Please review

**Review by sakshi-goyal-razorpay — APPROVED** [review]

**Comment by sakshi-goyal-razorpay** [comment]
LGTM

**Pull request merged**